README.md

```diff
@@ -20,6 +20,62 @@ Note the address bar indicating google.com. Also, since the content of USA Today
 - Can't request permissions
 - Can't be added to homescreen
 
+## Goals and non-goals
+
+### Goals
+
+* **authenticated-sharing**: Alice can download a package containing
+  a [PWA](https://developers.google.com/web/progressive-web-apps/checklist) from
+  origin `O`, and transmit it to their peer Bob, and then Bob can install
+  the PWA with a proof that it came from `O`.
+* **streamed-loading**: When a package is downloaded over TLS from its original
```

Comment:
Done, especially given the fact that TLS doesn't guarantee resources aren't dropped.